src/optimise/Minuit.cpp

```diff
@@ -204,3 +211,14 @@ Minuit::GetFitResult() const{
     return fFitResult;
     
 }
+
+DenseMatrix
+Minuit::CalcCovarianceMatrix(ROOT::Minuit2::FunctionMinimum functionMin) {
+    ROOT::Minuit2::MnUserCovariance minCovariance = functionMin.UserCovariance();
+    const std::vector<double>& minCovarianceVector = minCovariance.ROOT::Minuit2::MnUserCovariance::Data();
```

Comment:
I think it's the same (as far as I can tell it is). I can change it to the latter if you'd prefer. (Looking back on it, I'm not sure why I didn't just do that.)